src/dotnet/Users.Service/Module/UsersServiceModule.cs

```diff
@@ -144,6 +144,8 @@ public override void InjectServices(IServiceCollection services)
         fusion.AddComputeService<IChatReadPositions, ChatReadPositions>();
         fusion.AddComputeService<IServerKvas, ServerKvas>();
         fusion.AddComputeService<IServerKvasBackend, ServerKvasBackend>();
+        fusion.AddComputeService<IRecents, Recents>();
```

Comment:
`IRecentEntries`, etc.?

Reply:
The reason I am a bit in favor of renaming this is that `Recent` is kinda super generic & very short.

Reply:
done